Lib/test/coding20731.py

```diff
@@ -1,4 +1 @@
 #coding:latin1

```

Comment:
I don't think this one should be changed.

Reply:
You are right. The make patchcheck gave some problems for me, so I executed some scripts manually. I guess that triggered the changes. I will revert them